audit-service/src/main/java/uk/gov/gchq/palisade/service/audit/service/ServiceName.java

```diff
@@ -22,10 +22,21 @@
  * @since 0.4.0
  */
 public enum ServiceName {
-    AUDIT_SERVICE,
-    DATA_SERVICE,
-    PALISADE_SERVICE,
-    POLICY_SERVICE,
-    RESOURCE_SERVICE,
-    USER_SERVICE
+    ATTRIBUTE_MASKING_SERVICE("attribute-masking-service"),
+    DATA_SERVICE("data-service"),
+    FILTERED_RESOURCE_SERVICE("filtered-resource-service"),
+    PALISADE_SERVICE("palisade-service"),
+    POLICY_SERVICE("policy-service"),
+    RESOURCE_SERVICE("resource-service"),
+    TOPIC_OFFSET_SERVICE("topic-offset-service"),
+    USER_SERVICE("user-service");
```

Comment:
Attribute masking service, topic offset service and palisade service are all unused, are they required to be in here?

Reply:
They are not required but they should probably be left in as a full list of the services in Palisade in case there is any need for them in the future.
The data and filtered-resource services are the only ones used in the service implementation logic and the user service has been used in multiple tests.